<?xml version="1.0"?>
<launch>
    <node name="skeleton_to_3d" pkg="skeleton3d" type="skeleton_creator" output="screen">
      <param name="input_pose" type="str" value="/pose_estimator/pose" />
      <param name="input_pointcloud" type="str" value="/myxtion/depth_registered/points" />
      <param name="output_skeleton" type="str" value="/skeleton_to_3d/skeletons" />
      <param name="scatter_distance" type="int" value="8" />
    </node>

    <node name="skeleton_from_camera" pkg="skeleton3d" type="skeleton_visualiser" output="screen">
      <param name="input_skeleton" type="str" value="/skeleton_to_3d/skeletons" />
      <param name = "output_marker" type="str" value="/skeleton_to_3d_vis/skeleton_vis" />
    </node>
    
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find skeleton3d)/rviz/Skeleton_markers.rviz" />

</launch>
